.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+name: CI
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)